src/Apps/Settings/Routes/MyCollection/Components/MyCollectionEmptyState.tsx

```diff
@@ -7,230 +7,150 @@ import {
   Image,
   Message,
   ResponsiveBox,
+  Spacer,
   Text,
 } from "@artsy/palette"
 import { useMyCollectionTracking } from "Apps/MyCollection/Routes/Hooks/useMyCollectionTracking"
-import { EmptyMyCollectionPageProps } from "Apps/Settings/Routes/MyCollection/Components/EmptyMyCollectionPage"
+import { useAuthDialog } from "Components/AuthDialog"
 import { ModalType } from "Components/Authentication/Types"
 import { RouterLink } from "System/Router/RouterLink"
 import { useSystemContext } from "System/SystemContext"
 import { useFeatureFlag } from "System/useFeatureFlag"
-import { openAuthModal } from "Utils/openAuthModal"
 import { resized } from "Utils/resized"
-import { Media } from "Utils/Responsive"
 
-const DesktopLayout: React.FC<EmptyMyCollectionPageProps> = ({
-  loggedOutState,
-}) => {
+export const MyCollectionEmptyState: React.FC = () => {
   const isCollectorProfileEnabled = useFeatureFlag("cx-collector-profile")
 
-  const { mediator } = useSystemContext()
+  const { isLoggedIn } = useSystemContext()
+
   const {
     addCollectedArtwork: trackAddCollectedArtwork,
   } = useMyCollectionTracking()
 
   const image = resized(
     "https://files.artsy.net/images/my-coll-get-app-img.jpg",
-    {
-      width: 770,
-      height: 652,
-    }
+    { width: 910, height: 770 } // TODO: Source image should be 4:3, ideally. Is currently 709:600
   )
 
+  const { showAuthDialog } = useAuthDialog()
+
   return (
-    <GridColumns mb={12} gridRowGap={4} alignItems="center">
-      <Column span={6}>
-        <Text variant="xxl">Manage Your Art Collection Online</Text>
+    <GridColumns gridRowGap={4} alignItems="center">
+      <Column span={6} order={[1, 0]} py={[0, 2]}>
```

Comment:
Got rid of the entire `Mobile/Desktop` layout split with this one call to `order` — we should know about this property and use it instead of spinning up parallel layouts that are difficult to maintain.

Reply:
Added some vertical padding here but this component is inadequately specified at medium breakpoints in Figma.